src/main/java/io/deephaven/csv/parsers/Parser.java

```diff
@@ -75,6 +75,10 @@ long tryParse(
             throws CsvReaderException;
 
     class GlobalContext {
+        /**
+         * The 1-based column number that the parser is working on.
+         */
+        public final int colNum;
```

Comment:
Better to make this private and a getter for it IMO; or be consistent and let nullChunk be public.

Reply:
done